doc/classes/Area2D.xml

```diff
@@ -25,7 +25,21 @@
 		<method name="get_overlapping_bodies" qualifiers="const">
 			<return type="Node2D[]" />
 			<description>
-				Returns a list of intersecting [PhysicsBody2D]s. The overlapping body's [member CollisionObject2D.collision_layer] must be part of this area's [member CollisionObject2D.collision_mask] in order to be detected.
+				Returns a list of intersecting [PhysicsBody2D]s and [TileMap]s. The overlapping body's [member CollisionObject2D.collision_layer] must be part of this area's [member CollisionObject2D.collision_mask] in order to be detected.
+				For performance reasons (collisions are all processed at the same time) this list is modified once during the physics step, not immediately after objects are moved. Consider using signals instead.
+			</description>
+		</method>
+		<method name="has_overlapping_areas" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] when intersecting any [Area2D]s, otherwise [code]false[/code]. The overlapping area's [member CollisionObject2D.collision_layer] must be part of this area's [member CollisionObject2D.collision_mask] in order to be detected.
+				For performance reasons (collisions are all processed at the same time) the list of overlapping areas is modified once during the physics step, not immediately after objects are moved. Consider using signals instead.
+			</description>
+		</method>
+		<method name="has_overlapping_bodies" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] when intersecting any [PhysicsBody2D] or [TileMap], otherwise [code]false[/code]. The overlapping body's [member CollisionObject2D.collision_layer] must be part of this area's [member CollisionObject2D.collision_mask] in order to be detected.
```

Comment:
```suggestion
				Returns [code]true[/code] if intersecting any [PhysicsBody2D] or [TileMap], otherwise returns [code]false[/code]. The overlapping body's [member CollisionObject2D.collision_layer] must be part of this area's [member CollisionObject2D.collision_mask] in order to be detected.
```